symengine/ntheory.h

```diff
@@ -162,13 +162,19 @@ void nthroot_mod_list(std::vector<RCP<const Integer>> &roots, const RCP<const In
         const RCP<const Integer> &n, const RCP<const Integer> &m);
 //! A solution to x**n == a mod m. Return false if none exists.
 bool nthroot_mod(const Ptr<RCP<const Integer>> &root, const RCP<const Integer> &a,
-        const RCP<const Integer> &n, const RCP<const Integer> &m);
+        const RCP<const Integer> &n, const RCP<const Integer> &mod);
```

Comment:
I think we should keep the definition in `.h` the same as the implementation in `.cpp`. I think that's the least amount of confusion. So we should either change both the definition and the implementation, or the docstring.